api-samples/webRequest/background.js

```diff
@@ -0,0 +1,10 @@
+// Copyright 2021 Google LLC
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+// Alerts the user when an authorization request is requested via the console
+chrome.webRequest.onAuthRequired.addListener((details, callbackFunction) => {
+  console.log('An authorization request has been detected');
```

Comment:
I think it would be nice to actually detect a domain here (https://jigsaw.w3.org/HTTP/Basic/ is probably a good one), and pass some credentials.

We can then explain in the README (we should make one!) that installing this extension should allow you to visit that site without a prompt.